CONTRIBUTING.md

```diff
@@ -38,3 +38,120 @@ are conflicts (even trivially resolvable ones).
+- A "Contributor" is a person who wishes to provide a patch, being a set of commits that solve some clearly identified problem.
+- A "Maintainer" is a person who merges patches to the project. Maintainers are not developers; their job is to enforce process.
+- Contributors SHALL NOT have commit access to the repository unless they are also Maintainers.
+- Maintainers SHALL have commit access to the repository.
+- Everyone, without distinction or discrimination, SHALL have an equal right to become a Contributor under the terms of this contract.
+
+### Licensing and Ownership
+
+- The project SHALL use a share-alike license, such as BSD-3, the GPLv3 or a variant thereof (LGPL, AGPL), or the MPLv2.
+- All contributions to the project source code ("patches") SHALL use the same license as the project.
+- All patches are owned by their authors. There SHALL NOT be any copyright assignment process.
+- The copyrights in the project SHALL be owned collectively by all its Contributors.
+- Each Contributor SHALL be responsible for identifying themselves in the project Contributor list.
+
+### Patch Requirements
+
+- Maintainers MUST have a Platform account and SHOULD use their real names or a well-known alias.
+- Contributors SHOULD have a Platform account and MAY use their real names or a well-known alias.
+- A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
+- A patch MUST adhere to the code style guidelines of the project if these are defined.
+- A patch MUST adhere to the "Evolution of Public Contracts" guidelines defined below.
+- A patch SHALL NOT include non-trivial code from other projects unless the Contributor is the original author of that code.
+- A patch MUST compile cleanly and pass project self-tests on at least the principle target platform.
+- A patch commit message SHOULD consist of a single short (less than 50 character) line summarizing the change, optionally followed by a blank line and then a more thorough description.
+- A "Correct Patch" is one that satisfies the above requirements.
+
+### Development Process
+
+- Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+- To request changes, a user SHOULD log an issue on the project Platform issue tracker.
+- The user or Contributor SHOULD write the issue by describing the problem they face or observe.
+- The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
+- Users SHALL NOT log feature requests, ideas, or suggestions unrelated to Monero code or Monero's dependency code or Monero's potential/future dependency code or research which successfully implements Monero.
+- Users SHALL NOT log any solutions to problems (verifiable or hypothetical) of which are not explicitly documented and/or not provable and/or cannot be reasonbly proven.
```

Comment:
remove the 'of' after the close of the parentheses

Reply:
Link/resource proving that it's grammatically incorrect?

Reply:
It's obviously incorrect. Just like the misspelled "reasonbly" at the end of the sentence.

Reply:
>s/of//

I know you two are across the pond, just asking for clarification since there are differences between British and American English.

>s/reasonbly/reasonably/

Yes, well the perils of doing *everything* with vim.

Reply:
Whenever confronted with this type of situation that you're uncertain of, break it down and eliminate all the prepositional phrases to simplify the sentence and see if what's left still makes sense:

> Users shall not log any solutions to problems

Yes, the above makes sense, but now there's a whole ton of extraneous text that didn't contribute to the sentence at all.

> Users shall not log any solutions to problems which are not explicitly documented

Makes a whole lot more sense, and actually uses all of the content.

Reply:
>you're uncertain of

I was never uncertain.

>>http://www.grammar-quizzes.com/clauses-8.html#ofwhichclause
>>Of which replaces the subject —genitive noun or pronoun— in the modifying clause, which is placed after the inanimate noun that it modifies.

>solutions to problems (verifiable or hypothetical)
>problems
>^ inanimate noun that is modified

Quit wasting my time with grammar NACKs that are provably correct.

Reply:
You're really going to raise a fight over such a stupid point?

You're wrong.

"to problems" is neither the subject nor the object of this clause. "solutions" is. Get over yourself. If you're going to pick a fight over grammar, at least learn how to diagram sentence first.